src/modules/commander/commander_helper.h

```diff
@@ -56,6 +56,8 @@ bool is_vtol(const vehicle_status_s &current_status);
 bool is_vtol_tailsitter(const vehicle_status_s &current_status);
 bool is_fixed_wing(const vehicle_status_s &current_status);
 bool is_ground_rover(const vehicle_status_s &current_status);
+bool is_boat(const vehicle_status_s &current_status);
+bool is_ground_vehicle(const vehicle_status_s &current_status);
```

Comment:
Is there a reason we would want to distinguish between a boat and a rover? Wouldn't be better to simply replace `is_ground_rover` with `is_ground_vehicle`?

Reply:
Probably, I think this helper can be simplified a lot more since it was used earlier but most usages are not there anymore as far as I've seen.